sheet_to_triples/rdf.py

```diff
@@ -152,6 +152,10 @@ def normalise_model(model, ns, verbose):
                 s=_n3(key[0], ns), p=_n3(key[1], ns), o=by_key[key]['obj']))
         by_key[key] = t
 
+    # set can't be written as JSON and we don't need this any more, so get rid
+    # of it
+    del model['non_uniques']
```

Comment:
Yeah, this is a bit awkward, have put up alternative approach in this branch:
https://github.com/VisualMeaning/sheet-to-triples/pull/39